src/NpmUtilities.js

```diff
@@ -134,6 +134,11 @@ export default class NpmUtilities {
   }
 
   static getTagOpts(registry) {
-    return registry ? {env: {npm_config_registry: registry}} : null;
+    const opts = {};
+    if (registry) {
+      opts.env = Object.assign({}, process.env, {npm_config_registry: registry});
```

Comment:
I think creating a clone of the env as a base is a good thing in this case since we only need the extended params for tagging. If it's an overkill, or in fact a bad thing, please let me know!